apps/frontend/src/renderer/components/github-prs/components/PRFilterBar.tsx

```
@@ -0,0 +1,357 @@
+          <Search className="absolute left-2.5 top-1/2 -translate-y-1/2 h-4 w-4 text-muted-foreground" />
+          <Input
+            placeholder={t('prReview.searchPlaceholder')}
+            value={filters.searchQuery}
+            onChange={(e) => onSearchChange(e.target.value)}
+            className="h-8 pl-9 bg-background/50 focus:bg-background transition-colors"
+          />
+          {filters.searchQuery && (
+            <button
+              onClick={() => onSearchChange('')}
+              className="absolute right-2.5 top-1/2 -translate-y-1/2 text-muted-foreground hover:text-foreground"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)